src/main/java/space/obminyashka/items_exchange/dto/AdvertisementModificationDto.java

```diff
@@ -25,7 +25,6 @@
 
     @Positive(groups = Exist.class, message = "{invalid.exist.id}")
     @Zero(groups = New.class, message = "{new.advertisement.id.not-zero}")
-    @NotNull(message = "{invalid.not-null}")
```

Comment:
We use this DTO also when create adv.
Its like a validation. 
How will we notify front about incorrect value
 without it?